tests/subsys/net/lib/aws_fota/aws_fota_json/src/main.c

```diff
@@ -146,7 +230,12 @@ void test_main(void)
 			 ztest_unit_test(test_update_job_exec_rsp),
```

Comment:
A nit really, I'd consider keeping the tests on the list in the order they show up in the file. Up to you if you want to fix it or not.

Reply:
Think I've cleaned up this now.